config/cluster.json.sample

```diff
@@ -61,7 +61,7 @@
       {
         "name": "Elasticity Bootstrap Action",
         "scriptBootstrapAction": {
-          "path": "s3://snowplow-hosted-assets-us-east-1/common/emr/snowplow-ami4-bootstrap-0.2.0.sh",
+          "path": "s3://snowplow-hosted-assets-us-east-1/common/emr/snowplow-ami5-bootstrap-0.1.0.sh",
```

Comment:
Is this still needed?

Reply:
🤷

The purpose of `cluster.json.sample` is to demo the format of the configuration file.  So whether it's needed or not, there should be an example of a bootstrap action.  This one seemed reasonable, because we own it and it's publicly visible.

Do you have any suggestions for an alternative bootstrap action?  I guess I could make it fictious, like `s3://my-bucket/path/to/bootstrap.sh`